pkg/server/plugin/nodeattestor/aws/iid.go

```diff
@@ -77,13 +77,13 @@ type IIDAttestorPlugin struct {
 
 // IIDAttestorConfig holds hcl configuration for IID attestor plugin
 type IIDAttestorConfig struct {
-	caws.SessionConfig   `hcl:",squash"`
-	SkipBlockDevice      bool   `hcl:"skip_block_device"`
-	SkipEC2AttestCalling bool   `hcl:"skip_ec2_attest_calling"`
-	AgentPathTemplate    string `hcl:"agent_path_template"`
-	pathTemplate         *template.Template
-	trustDomain          string
-	awsCaCertPublicKey   *rsa.PublicKey
+	caws.SessionConfig `hcl:",squash"`
+	SkipBlockDevice    bool     `hcl:"skip_block_device"`
+	AllowList          []string `hcl:"allow_list"`
```

Comment:
nit: would be good to make this more descriptive - something like `locally_validated_account_ids` or `local_validation_whitelist`